macro/Version.py

```diff
@@ -36,16 +55,40 @@ def execute(macro, args):
                     'New in %s</span>' % version)
 
     def distro_html(distro, distros):
-        active = [distro.encode("iso-8859-1")]
-        inactive = [x.encode("iso-8859-1") for x in distros if not x == distro]
+        html = ""
```

Comment:
I don't think it changes the scope; I had that there for debugging by adding more stuff to the html in the loop. I'll move it, but I don't see that it matters.